src/pages/partners/payment-agent/_p2p_banner.tsx

```diff
@@ -108,7 +107,6 @@ const P2PBanner = () => {
                     <LeftChild>
                         <StyledHeader
                             as="h3"
-                            type="heading-3"
```

Comment:
Why are we removing the `type`?

Reply:
Because while we zoom up the page a font-size also grow because use rem as default. If we have any other option please let me know 🙏

Reply:
Please extend a regular `h3` instead of `Header` here then.

Reply:
Realized as new styled component: `const StyledH3 = styled.h3`
    font-weight: bold;
    color: white;
    font-size: 32px;
    line-height: 40px;
    margin-bottom: 16px;
    @media ${device.tabletL} {
        font-size: 32px;
        line-height: 40px;
        max-width: 378px;
    }`
    Please check 🙏